contracts/cw1/Cargo.toml

```diff
@@ -12,4 +12,8 @@ homepage = "https://cosmwasm.com"
 cosmwasm-std = { version = "1.1", features = ["staking"] }
 cosmwasm-schema = "1.1"
 serde = { version = "1", default-features = false, features = ["derive"] }
-sylvia = { version = "0.2.2", path = "../../sylvia" }
+sylvia = { version = "0.2.2", path = "../../sylvia", features = ["mt"] }
```

Comment:
We don't want Sylvia to have "mt" in a normal build. Add it to dev-dependencies, and add an additional feature "mt" feature which transitively enables "mt" feature for Sylvia.

Reply:
As for example this is only part of implementation containing helpers for traits.
There will be a second delivery with the multitest helpers generation for contracts. Then there will be examples.